README.md

```diff
@@ -1,20 +1,51 @@
-# aspen-protein-viewer
+# Aspen-Protein-Viewer
 
-## About
+Component to render the 3D structure of the target.
```

Comment:
Component to render the 3D structure of the protein from RCSB PDB (Protein Data Bank).